lib/open_api_spex/operation_dsl.ex

```diff
@@ -0,0 +1,57 @@
+defmodule OpenApiSpex.OperationDsl do
+  alias OpenApiSpex.OperationBuilder
+
+  defmacro operation(action, spec) do
+    operation_def(action, spec)
+  end
+
+  defmacro tags(tags) do
```

Comment:
It might be useful to allow `security` to be defined like `tags` at the module level, and have it apply to all operations in the controller.